Sources/tuist/tuist.docc/Tutorials/01-Getting-Started/01-01-Install-Uninstall-Tuist/01-01-01-01.txt

```diff
@@ -0,0 +1 @@
+% curl -Ls https://install.tuist.io | bash
```

Comment:
not sure if DocC is imposing the filenames, but if not I would use meaningful names (e.g. this should be install tuist or something similar)

Reply:
So, the issue with names is it gets confusing quite fast. You need to always keep in mind to have unique names. Also, you still might have to append numbers at the end, when you are still on the same topic.
This just follows a simple system of nth chapter, nth tutorial, nth section and nth code block.

We use this format for our internal documentation and it helps us group all related code effectively. But yeah, I get your point, the naming can be quite abstract at first.

Reply:
If you have to add one file in the middle, you should rename all the following ones which I don't think scales well.
If you use descriptive names you should never end up with the same name, unless the file has the same content, isn't it?

Reply:
That is indeed a valid point, I'll rename them then.